docs/topics/http/urls.txt

```diff
@@ -208,13 +206,13 @@ A convenient trick is to specify default parameters for your views' arguments.
 Here's an example URLconf and view::
 
     # URLconf
-    from django.conf.urls import url
+    from django.urls import path
 
     from . import views
 
     urlpatterns = [
-        url(r'^blog/$', views.page),
-        url(r'^blog/page(?P<num>[0-9]+)/$', views.page),
+        path('blog/', views.page),
+        path('blog/page<int:num>/', views.page),
     ]
 
     # View (in blog/views.py)
```

Comment:
The unchanged parts below still contain references to regular expressions (which is fine), but might need a bit of rewording.